go/oasis-node/cmd/common/common.go

```diff
@@ -200,3 +202,15 @@ func LoadEntity(dataDir string) (*entity.Entity, signature.Signer, error) {
 	entitySignerFactory := fileSigner.NewFactory(dataDir, signature.SignerEntity)
 	return entity.Load(dataDir, entitySignerFactory)
 }
+
+func LoadSigner(signer string) (signature.Signer, error) {
+	switch signer {
+	case "ledger":
```

Comment:
These should be constants in each signer package, e.g. `ledgerSigner.SignerName`.